TenureInformationApi.Tests/V1/Boundary/Validation/EditTenureInformationRequestValidatorTests.cs

```diff
@@ -0,0 +1,73 @@
+using FluentValidation.TestHelper;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TenureInformationApi.V1.Boundary.Requests;
+using TenureInformationApi.V1.Boundary.Requests.Validation;
+using Xunit;
+
+namespace TenureInformationApi.Tests.V1.Boundary.Validation
+{
+    public class EditTenureInformationRequestValidatorTests
+    {
+        public EditTenureDetailsRequestValidation _classUnderTest;
+
+        public EditTenureInformationRequestValidatorTests()
+        {
+            _classUnderTest = new EditTenureDetailsRequestValidation();
+        }
+
+        /*
+
+        These tests are the same in CreateTenureRequestValidator.
+        Is there a clean way to reuse these tests?
+
+        */
```

Comment:
Not really.
But as the start date is optional (implied by the DynamoDbGateway implementation) surely they are not completely the same. So should there also be tests for if the start date isn't passed?